gui/create-item.lua

```diff
@@ -184,11 +211,176 @@ local function getCreatureRaceAndCaste(caste)
  return df.global.world.raws.creatures.list_creature[caste.index],df.global.world.raws.creatures.list_caste[caste.index]
 end
 
+function createCorpsePiece(creator, bodypart, partlayer, creatureID, casteID, generic, quality) -- this part was written by four rabbits in a trenchcoat (ppaawwll)
+ -- (partlayer is also used to determine the material if we're spawning a "generic" body part (i'm just lazy lol))
```

Comment:
yeah honestly adding a separate variable just for the material for a generic item is a bit, excessive? it's not used otherwise if it's not generic so, /shrug